jsk_arc2017_baxter/euslisp/lib/stow-interface.l

```diff
@@ -168,10 +168,10 @@
                   :do-stop-grasp nil
                   :grasp-style grasp-style-))
       (when moveit-p- (send self :add-tote-scene arm))
-      ;; Don't trust pressure sensor
-      ;; (unless (eq pick-result :ik-failed)
-      ;;   (setq graspingp (send *ri* :graspingp arm grasp-style-)))
-      ;; (unless graspingp (return-from :pick-object nil))
+      ;; we trust pressure sensor
+      (unless (eq pick-result :ik-failed)
+        (setq graspingp (send *ri* :graspingp arm grasp-style-)))
+      (unless graspingp (return-from :pick-object nil))
```

Comment:
This is my mistake, but please put this `(unless graspingp (return-from :pick-object nil))` into `unless (eq pick-result :ik-failed)`